storage/fuzz/fuzz_targets/current_ordered_operations.rs

```diff
@@ -0,0 +1,300 @@
+#![no_main]
+
+use arbitrary::Arbitrary;
+use commonware_cryptography::Sha256;
+use commonware_runtime::{buffer::PoolRef, deterministic, Runner};
+use commonware_storage::{
+    adb::current::{ordered::Current, Config},
+    mmr::{hasher::Hasher as MmrHasher, Location, StandardHasher as Standard},
+    translator::TwoCap,
+};
+use commonware_utils::{sequence::FixedBytes, NZUsize, NZU64};
+use libfuzzer_sys::fuzz_target;
+use std::collections::HashMap;
+
+type Key = FixedBytes<32>;
+type Value = FixedBytes<32>;
+type RawKey = [u8; 32];
+type RawValue = [u8; 32];
+
+#[derive(Arbitrary, Debug, Clone)]
+enum CurrentOperation {
+    Update { key: RawKey, value: RawValue },
+    Delete { key: RawKey },
+    Get { key: RawKey },
+    Commit,
+    Prune,
+    OpCount,
+    Root,
+    RangeProof { start_loc: u64, max_ops: u64 },
+    KeyValueProof { key: RawKey },
+    GetSpan { key: RawKey },
+    ExclusionProof { key: RawKey },
+}
+
+#[derive(Arbitrary, Debug)]
+struct FuzzInput {
+    operations: Vec<CurrentOperation>,
+}
+
+const PAGE_SIZE: usize = 88;
+const PAGE_CACHE_SIZE: usize = 8;
+
+fn fuzz(data: FuzzInput) {
+    let runner = deterministic::Runner::default();
+
+    runner.start(|context| async move {
+        let mut hasher = Standard::<Sha256>::new();
+        let cfg = Config {
+            mmr_journal_partition: "fuzz_current_mmr_journal".into(),
+            mmr_metadata_partition: "fuzz_current_mmr_metadata".into(),
+            mmr_items_per_blob: NZU64!(11),
+            mmr_write_buffer: NZUsize!(1024),
+            log_journal_partition: "fuzz_current_log_journal".into(),
+            log_items_per_blob: NZU64!(7),
+            log_write_buffer: NZUsize!(1024),
+            bitmap_metadata_partition: "fuzz_current_bitmap_metadata".into(),
+            translator: TwoCap,
+            buffer_pool: PoolRef::new(NZUsize!(PAGE_SIZE), NZUsize!(PAGE_CACHE_SIZE)),
```

Comment:
nit: some magic numbers are consts but not all?

Reply:
Copy/pasta... changed here & in the source from which it was copied to use `const` consistently.